pyproject.toml

```diff
@@ -1,5 +1,6 @@
 [build-system]
-requires = ["setuptools", "wheel"]
+requires = ["setuptools"]
```

Comment:
setuptools' docs used to incorrectly say you should include `wheel`. They've corrected this.